service/discovery/extra/csaf/document_properties.go

```diff
@@ -31,7 +33,40 @@ func transportEncryption(state *tls.ConnectionState) (te *ontology.TransportEncr
 		}
 
 		te.Protocol = constants.TLS
+		cs := cipherSuite(state.CipherSuite)
+		if cs != nil {
+			te.CipherSuites = append(te.CipherSuites, cs)
+		}
 	}
 
 	return te
 }
+
+// cipherSuite builds a [ontology.CipherSuite] object out of the cipher suite identifier of the tls package, e.g.
+// [tls.TLS_AES_128_GCM_SHA256].
+func cipherSuite(id uint16) *ontology.CipherSuite {
+	if id == tls.TLS_AES_128_GCM_SHA256 {
+		return &ontology.CipherSuite{
+			SessionCipher: "AES-128-GCM",
+			MacAlgorithm:  "SHA-256",
+		}
+	} else if id == tls.TLS_AES_256_GCM_SHA384 {
+		return &ontology.CipherSuite{
+			SessionCipher: "AES-256-GCM",
```

Comment:
maybe define constants for them if we use them more often. Maybe there are constants in some packages, e.g. in crypto?